packages/kit/src/core/sync/write_root.js

```diff
@@ -57,7 +55,7 @@ export function write_root(manifest_data, output) {
 					setContext('__svelte__', stores);
 				}
 
-				$: stores.page.set(page);
+				$: stores.page.set({...page, form});
```

Comment:
A change more in line with the how it's done for the other values of page (namely `data`) would be to adjust the creation of the `page` object. This would need adjustments to `runtime/server/page/render.js` and `runtime/client.js` where that object is created.

Reply:
Is it even possible to automatically update `$page.form` automatically on the client. As I understand it, forms do not have dynamic behavior on the client by default. The only thing I could do here is to update the `enhance` function to also update the `form` field.

Could you maybe elaborate a bit more on what the client should do here?